warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -78,3 +78,63 @@ models:
           expression: "is_implemented"
     columns:
       - name: is_implemented
+
+  - name: fct_daily_reports_service
+    description: |
+      Table consolidating daily service hours, routes, and stops by feed,
+      filtered to the specific feed used for each monthly report.
+      Replaces complex Siuba code in reports notebook and allows for
+      visibility into service hour changes before the report generation process.
+      Primary key is composite of service_date, calitp_itp_id, calitp_url_number.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - service_date
+            - calitp_itp_id
+            - calitp_url_number
+    columns:
+      - name: service_date
+        description: Date on which this service level was present
+        tests:
+          - not_null
+      - name: feed_key
+        tests:
+          - not_null
+        description: feed_key only present with active service on date
+      - name: ttl_service_hours
+        description: Total service hours across entire feed on date, if any
+      - name: n_trips
+        description: Total number of trips ran on date, if any
+      - name: first_departure_ts
+        description: Time of first departure on date
+      - name: last_departure_ts
+        description: Time of last departure on date
+      - name: n_stops
```

Comment:
Change this to `n_stop_times` and description should say number of scheduled stop events